packages/grid/x-data-grid-pro/src/tests/exportPrint.DataGrid.test.tsx

```diff
@@ -0,0 +1,262 @@
+import * as React from 'react';
+import { DataGrid, GridToolbarExport, GridRowsProp, GridColumns } from '@mui/x-data-grid';
+import { createRenderer, screen, fireEvent } from '@mui/monorepo/test/utils';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+describe('<DataGridPro /> - Export - Print', () => {
+  const { render, clock } = createRenderer({ clock: 'fake' });
+
+  const rows: GridRowsProp = [
+    { id: 1, col1: '1-1', col2: '1-2' },
```

Comment:
Could you use `getData` or `useData` instead? I think we were avoiding hardcoded data in tests.

https://github.com/mui/mui-x/blob/acb00d2005b3855de60b82e8b39d3325f8b21b50/packages/grid/x-data-grid/src/tests/selection.DataGrid.test.tsx#L37